cardano-crypto-class/src/Cardano/Crypto/KES/Class.hs

```diff
@@ -261,3 +266,19 @@ encodeSignedKES (SignedKES s) = encodeSigKES s
 
 decodeSignedKES :: KESAlgorithm v => Decoder s (SignedKES v a)
 decodeSignedKES = SignedKES <$> decodeSigKES
+
+--
+-- 'Size' expressions
+--
+
+accessProxy :: Proxy (f v) -> Proxy v
+accessProxy _ = Proxy
+
+encodedVerKeyKESSizeExpr :: KESAlgorithm v => Proxy (VerKeyKES v) -> Size
+encodedVerKeyKESSizeExpr proxy = fromIntegral (sizeVerKeyKES (accessProxy proxy))
+
+encodedSignKeyKESSizeExpr :: KESAlgorithm v => Proxy (SignKeyKES v) -> Size
+encodedSignKeyKESSizeExpr proxy = fromIntegral (sizeSignKeyKES (accessProxy proxy))
+
+encodedSigKESSizeExpr :: KESAlgorithm v => Proxy (SigKES v) -> Size
+encodedSigKESSizeExpr proxy = fromIntegral (sizeSigKES (accessProxy proxy))
```

Comment:
Again, could do with comments. This is clever tricky stuff, with subtle types.

Reply:
This is just using `Num` instance.  The reason for this is that `ToCBOR` module does not export `Size` constructors, it forces to either use `Num` instance or `NumDecimals` extension.